habitat_baselines/run.py

```diff
@@ -0,0 +1,18 @@
+from habitat_baselines.common.base_trainer import get_trainer
+from habitat_baselines.common.utils import experiment_args, get_exp_config
+
+
+def main():
```

Comment:
I really like simplicity of the entry point and how easy we can contribute more baselines without messing other ones.